ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -438,6 +460,36 @@ zipOverLedgerTablesTicked f l tables2 =
       (\tables1 -> zipLedgerTables f tables1 tables2)
       l
 
+class TableStuff l => SufficientSerializationForAnyBackingStore (l :: LedgerStateKind) where
+  codecLedgerTables :: LedgerTables l CodecMK
+  codecLedgerTables = pureLedgerTables $ emptyAppliedMK SCodecMK
+
+valuesMKEncoder :: SufficientSerializationForAnyBackingStore l => LedgerTables l ValuesMK -> CBOR.Encoding
+valuesMKEncoder tables =
+       CBOR.encodeListLen (getSum (foldLedgerTables (\_ -> Sum 1) tables))
+    <> foldLedgerTables2 go codecLedgerTables tables
+  where
+    go :: ApplyMapKind CodecMK k v -> ApplyMapKind ValuesMK k v -> CBOR.Encoding
+    go (ApplyCodecMK encK encV _decK _decV) (ApplyValuesMK (UtxoValues m)) =
+         CBOR.encodeMapLen (fromIntegral $ Map.size m)
+      <> Map.foldMapWithKey (\k v -> CBOR.encodeListLen 2 <> encK k <> encV v) m
+
+valuesMKDecoder :: forall s l . SufficientSerializationForAnyBackingStore l => CBOR.Decoder s (LedgerTables l ValuesMK)
+valuesMKDecoder = do
+  numTables <- CBOR.decodeListLen
+  mapLen    <- CBOR.decodeMapLen
+  ret <- traverseLedgerTables (go mapLen) codecLedgerTables
+  Exn.assert ((getSum (foldLedgerTables (\_ -> Sum 1) ret)) == numTables)
+    $ return ret
+ where
+  go :: forall k v . Ord k
+     => Int
+     -> ApplyMapKind CodecMK k v
+     -> CBOR.Decoder s (ApplyMapKind ValuesMK k v)
+  go len (ApplyCodecMK _encK _encV decK decV) =
+        ApplyValuesMK . UtxoValues . Map.fromList
+    <$> sequence (replicate len (CBOR.decodeListLenOf 2 >> (,) <$> decK <*> decV))
```

Comment:
Minor: convention suggests `*>` here instead of `>>`.

Reply:
Applied suggestion